packages/docs/src/components/DocsNavigation.vue

```diff
@@ -71,10 +71,17 @@ export default {
     }
   },
   methods: {
+    updateCopyButton(icon, text) {
+      this.copyIcon = icon
+      this.copyText = text
+    },
     copy () {
-      this.$clipboard(this.code)
-      this.copyIcon = 'fa fa-check'
-      this.copyText = 'Copied'
+      try {
+        window.navigator.clipboard.writeText(this.code)
```

Comment:
According to specification https://w3c.github.io/clipboard-apis/#dom-clipboard-writetext, write text is an async function. You should use `await` here to catch errors.
```suggestion
        await window.navigator.clipboard.writeText(this.code)
```

Can you check if I correct? Maybe this `await` can break your code 😄